simulators/carla_simulator.py

```diff
@@ -26,7 +25,9 @@ class RecurrentState:
     It should not be modified, but rather passed along as received.
```

Comment:
Why are those dataclasses redefined instead of being imported from `invertedai`?